Allow for forcing index creation

Rua-Yuki opened the pull request:
This PR adds a config option which allows for forcing index creation in scenarios where it would otherwise be blocked.

Most notably, indexes can now be created regardless of model migration preference, provided the force-update flag is set.

The flag may be easily supplied via commandline option:
```bash
sails console --mongoat.forceUpdate
```

(readme has been [updated](https://github.com/fpm-git/sails-hook-mongoat/blob/dde660bd98f25cec5feb18ee29df4b43120de5f1/README.md#forcing-updates-to-run-in-safe-migration-mode) to reflect this new capability)

Rua-Yuki committed:
Add support for index force update config

Rua-Yuki committed:
Fix minor issues index.js

Rua-Yuki committed:
Update README with force update info

Maelstrom96 commented on index.js:
```diff
@@ -61,6 +62,44 @@ const SailsHookMongoat = (sails) => ({
+    if (sails.config.models && !['alter', 'drop'].includes(sails.config.models.migrate)) {
+      return {
+        shouldCreate: false,
+        forbiddenReason: `Model migration strategy "${sails.config.models.migrate}" was set. Use "alter" or "drop" to enable index creation, or build them manually.`,
```
Can you modify the message to say that it can be forced using config? (instead of "build them manually".

p-e-n-g-u replied:
Added some log about the force option anytime updates are blocked: https://github.com/fpm-git/sails-hook-mongoat/pull/1/commits/62b7592943471367a78e147e9c2bb1e8943a4097

Maelstrom96 commented:
Other than my little comment, everything looks good. Feel free to merge and do the release after this is resolved :)

Rua-Yuki committed:
Add note about forceUpdate when updates are blocked

Addresses https://github.com/fpm-git/sails-hook-mongoat/pull/1#discussion_r281758303
